distrib-dl

```diff
@@ -329,6 +332,35 @@ function download_fedora() {
     fi
 }
 
+function download_ubuntu() {
+    # Official netinstall image without firmwares (disabled)
+    #debian_base_url="https://cdimage.debian.org/debian-cd/current/${architecture}/iso-cd/"
+    #debian_iso_filename="debian-${debian_version}-${architecture}-netinst.iso"
+    # Unofficial image including firmwares
```

Comment:
please remove the 4 commented out lines above, there are no alternative images for ubuntu